src/main/features/claim/routes/eligibility/over-18-defendant.ts

```diff
@@ -0,0 +1,49 @@
+import * as express from 'express'
+
+import { Paths } from 'claim/paths'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { Form } from 'forms/form'
+import { Eligibility } from 'claim/form/models/eligibility/eligibility'
+import { FormValidator } from 'forms/validation/formValidator'
+import { User } from 'idam/user'
+import { DraftService } from 'services/draftService'
+import { NotEligibleReason } from 'claim/helpers/eligibility/notEligibleReason'
+import { ValidationGroups } from 'claim/helpers/eligibility/validationGroups'
+import { Draft } from '@hmcts/draft-store-client'
+import { DraftClaim } from 'drafts/models/draftClaim'
+import { DefendantAgeOption } from 'claim/form/models/eligibility/DefendantAgeOption'
```

Comment:
We agreed to keep file names camel-cased. To stick to it we should rename it to be `defendantAgeOption`.

REC